### ISSUE TITLE
Redmine 4.2.3 doesn't start after installing custom workflow plugin

### ISSUE THREAD

**Issue by mbrigl** [anteo/redmine_custom_workflows#248]
I migrated an older redmine (4.04) instance to redmine 4.2.3. The new installation works, but only without the custom_workflow plugin. Once I add the plugin (from master branch) the redmine doesn't start anymore.

Redmine runs on a Ubuntu 20.04 Server with Ruby 2.7.5 running. 

Did anyone have the same troubles? The error tells that *activemodel-serializers-xml 1.0.2* couldn't be find, but the module in installed.

~~~
 An error occurred while starting up the preloader.
  Error ID: 5944d469
  Error details saved to: /tmp/passenger-error-BBXgyi.html
  Message from application: <p>It looks like Bundler could not find a gem. Maybe you didn't install all the gems that this application needs. To install your gems, please run:</p>

  <pre class="commands">bundle install</pre>

<p>If that didn't work, then the problem is probably caused by your application being run under a different environment than it's supposed to. Please check the following:</p>

<ol>
  <li>Is this app supposed to be run as the <code>toladmin</code> user?</li>
  <li>Is this app being run on the correct Ruby interpreter? Below you will
      see which Ruby interpreter Phusion Passenger attempted to use.</li>
</ol>

<p>-------- The exception is as follows: -------</p>
Could not find activemodel-serializers-xml-1.0.2 in any of the sources (Bundler::GemNotFound)
<pre>  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/spec_set.rb:86:in `block in materialize&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/spec_set.rb:80:in `map!&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/spec_set.rb:80:in `materialize&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/definition.rb:170:in `specs&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/definition.rb:237:in `specs_for&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/definition.rb:226:in `requested_specs&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/runtime.rb:101:in `block in definition_method&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/runtime.rb:20:in `setup&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler.rb:149:in `setup&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/setup.rb:20:in `block in &lt;top (required)&gt;&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/ui/shell.rb:136:in `with_level&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/ui/shell.rb:88:in `silence&#39;
  /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/setup.rb:20:in `&lt;top (required)&gt;&#39;
  /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require&#39;
  /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require&#39;
  /usr/lib/ruby/vendor_ruby/phusion_passenger/loader_shared_helpers.rb:430:in `activate_gem&#39;
  /usr/lib/ruby/vendor_ruby/phusion_passenger/loader_shared_helpers.rb:297:in `block in run_load_path_setup_code&#39;
  /usr/lib/ruby/vendor_ruby/phusion_passenger/loader_shared_helpers.rb:435:in `running_bundler&#39;
  /usr/lib/ruby/vendor_ruby/phusion_passenger/loader_shared_helpers.rb:296:in `run_load_path_setup_code&#39;
  /usr/share/passenger/helper-scripts/rack-preloader.rb:100:in `preload_app&#39;
  /usr/share/passenger/helper-scripts/rack-preloader.rb:156:in `&lt;module:App&gt;&#39;
  /usr/share/passenger/helper-scripts/rack-preloader.rb:30:in `&lt;module:PhusionPassenger&gt;&#39;
  /usr/share/passenger/helper-scripts/rack-preloader.rb:29:in `&lt;main&gt;&#39;</pre>
~~~

**Comment by picman** [comment]
Have you run `bundle install` after you had added the plugin?

**Comment by mbrigl** [comment]
Yes, I did.

running the command I got:

~~~
gem search activemodel-serializers-xml

*** REMOTE GEMS ***

activemodel-serializers-xml (1.0.2)

~~~

**Comment by mbrigl** [comment]
Yes, I did run it in the root folder. Running it I got following output:
The installation didn't throw any error. Only accessing redmine, through the web, caused this error above.
~~~
The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x64-mingw32, x86-mswin32. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x64-mingw32 x86-mswin32`.
The dependency ffi (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x64-mingw32, x86-mswin32. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x64-mingw32 x86-mswin32`.
Using rake 13.0.6
Using concurrent-ruby 1.1.9
Using i18n 1.8.11
Using minitest 5.15.0
Using thread_safe 0.3.6
Using tzinfo 1.2.9
Using activesupport 5.2.6
Using builder 3.2.4
Using erubi 1.10.0
Using mini_portile2 2.5.3
Using racc 1.6.0
Using nokogiri 1.11.7 (x86_64-linux)
Using rails-dom-testing 2.0.3
Using crass 1.0.6
Using loofah 2.13.0
Using rails-html-sanitizer 1.4.2
Using actionview 5.2.6
Using rack 2.2.3
Using rack-test 1.1.0
Using actionpack 5.2.6
Using nio4r 2.5.8
Using websocket-extensions 0.1.5
Using websocket-driver 0.7.5
Using actioncable 5.2.6
Using globalid 1.0.0
Using activejob 5.2.6
Using mini_mime 1.0.3
Using mail 2.7.1
Using actionmailer 5.2.6
Using method_source 1.0.0
Using thor 1.2.1
Using railties 5.2.6
Using actionpack-xml_parser 2.0.1
Using activemodel 5.2.6
Using arel 9.0.0
Using activerecord 5.2.6
Using marcel 1.0.2
Using activestorage 5.2.6
Using public_suffix 4.0.6
Using addressable 2.8.0
Using bundler 2.1.4
Using chunky_png 1.4.0
Using css_parser 1.11.0
Using csv 3.1.9
Using htmlentities 4.3.4
Using mini_magick 4.11.0
Using net-ldap 0.17.0
Using pg 1.2.3
Using ruby-openid 2.9.2
Using rack-openid 1.4.2
Using sprockets 4.0.2
Using sprockets-rails 3.4.2
Using rails 5.2.6
Using rbpdf-font 1.19.1
Using rbpdf 1.20.1
Using redcarpet 3.5.1
Using request_store 1.5.1
Using roadie 4.0.0
Using roadie-rails 2.2.0
Using rotp 6.2.0
Using rouge 3.26.1
Using rqrcode_core 1.2.0
Using rqrcode 2.1.0
Using rubyzip 2.3.2
Bundle complete! 36 Gemfile dependencies, 64 gems now installed.
~~~
